introduction.md

```diff
@@ -0,0 +1 @@
+My name is Justin.
```

Comment:
Sure, I have committed a change saying that Elixir is my favourite Code School path.  Of course, I also have enjoyed the Git path, and the Ruby path, and am curious to try some others.